docs/en/guides/operational/adot-at-scale/adot-java-spring/adot-java-spring.md

```diff
@@ -0,0 +1,153 @@
+# Instrumenting Java Spring Applications
+
+This article describes an approach for manually instrumenting [Spring-Integration](https://spring.io/why-spring) applications utilizing [Open Telemetry](https://opentelemetry.io/) and [X-ray](https://aws.amazon.com/xray/).
+
+The Spring-integration framework is designed to enable the development of integration solutions typical of event-driven architectures and messaging-centric architectures. On the other hand, OpenTelemetry is much more focused on micro services architectures, in which services communicate and coordinate with each other using HTTP requests. Therefore this guide will provide a recommendation of how to instrument Spring-integration applications using “manual instrumentation” with the OpenTelemetry API.
+
+## Background Information
+
+### What is tracing?
+
+The following quote from the [OpenTelemetry documentation](https://opentelemetry.io/docs/concepts/signals/traces/) gives a good overview of what a trace's purpose is:
+
+!!! quote
+    Traces give us the big picture of what happens when a request is made to an application. Whether your application is a monolith with a single database or a sophisticated mesh of services, traces are essential to understanding the full “path” a request takes in your application.
+
+Given that one of the main benefits of tracing is end-to-end visibility of a request, it is important for traces to link properly all the way from the request origin to the backend. A common way of doing this in OpenTelemetry is to utilize [nested spans](https://opentelemetry.io/docs/instrumentation/java/manual/#create-nested-spans). This works in a microservices architecture where the spans are passed from service to service until they reach the final destination. In a Spring application, we need to create parent/child relationships between spans created both remotely AND locally.
+
+## Tracing Utilizing Context Propagation
+
+This time we are going to suggest a different approach: Instead of passing spans around the application code, we are going to use the messages in the application to transport the context from spans associated with the previous message. To achieve this we are going to use this [approach](https://opentelemetry.io/docs/instrumentation/java/manual/#context-propagation).
```

Comment:
This paragraph fells disconnected from the rest.

Reply:
I think you can direct to the point and say that we are going to use context propagation to manually instrument the application